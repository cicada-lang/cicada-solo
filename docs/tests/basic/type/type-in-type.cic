let t: Type = Type

compute t

let pair_of_types: Type = Pair(Type, Type)

compute pair_of_types
