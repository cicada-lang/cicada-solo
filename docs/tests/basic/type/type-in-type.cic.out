Type: Type
Pair(Type, Type): Type
